collections/_release_4_5/entry-platforms-windows-modify-metadata-without-rcedit.md

```diff
@@ -0,0 +1,20 @@
+---
+type: entry
+section: platforms
+subsection: windows
+rank: 0
+importance: 2
+anchor: modify-metadata-without-rcedit
+title: Modify metadata without `rcedit`
+blockquote: So long, `rcedit`!
+text: |
+  Windows `.exe` files metadata (such as a custom icon, the product name, and the company information) is stored internally as "resources".
+
+  For years, in order to modify these resources, the Windows exporter needed to access an external Windows program: [`rcedit`](https://github.com/electron/rcedit). That made it really difficult to export for Windows on a non-Windows machine.
+
+  That’s until now. The Windows exporter now knows how to edit the resources natively. It means you can edit Windows export metadata without a hitch, on every platform.
```

Comment:
```suggestion
  But now, the Windows exporter knows how to edit the resources natively, so you can edit Windows export metadata without a hitch on every platform.
```

Reply:
I think the first "edit" could be changed to maybe "handle", to reduce the repetition of "edit".